exonum-java-binding/core/rust/ejb-app/start_cryptocurrency_node.sh

```diff
@@ -37,41 +37,34 @@ echo "PROJ_ROOT=${EJB_ROOT}"
 
 header "PREPARE PATHS"
 
-CORE_TXT="core/target/ejb-core-classpath.txt"
 CRYPTOCURRENCY_TXT="cryptocurrency-demo/target/cryptocurrency-classpath.txt"
-EJB_CLASSPATH="$(cat ${EJB_ROOT}/${CORE_TXT}):$(cat ${EJB_ROOT}/${CRYPTOCURRENCY_TXT})"
-EJB_CLASSPATH="${EJB_CLASSPATH}:${EJB_ROOT}/core/target/classes"
+EJB_CLASSPATH="$(cat ${EJB_ROOT}/${CRYPTOCURRENCY_TXT})"
 EJB_CLASSPATH="${EJB_CLASSPATH}:${EJB_ROOT}/cryptocurrency-demo/target/classes"
 echo "EJB_CLASSPATH=${EJB_CLASSPATH}"
-EJB_LOG_CONFIG_PATH="${EJB_APP_DIR}/log4j2.xml"
+EJB_LOG_CONFIG_PATH="${EJB_APP_DIR}/log4j-fallback.xml"
 
-EJB_LIBPATH="${EJB_ROOT}/core/rust/target/debug"
-echo "EJB_LIBPATH=${EJB_LIBPATH}"
-RUST_LIB_DIR="$(rustup run 1.32.0 rustc --print sysroot)/lib"
-echo "RUST_LIB_DIR=${RUST_LIB_DIR}"
-
-export LD_LIBRARY_PATH="$EJB_LIBPATH:$RUST_LIB_DIR:$JVM_LIB_PATH"
+export LD_LIBRARY_PATH="$JVM_LIB_PATH"
```

Comment:
can be removed

Reply:
What exactly? @vitvakatu Isn't this var required to locate libjvm?

Reply:
Yep, exactly